.github/workflows/build-test-deploy.yml

```diff
@@ -8,9 +8,9 @@ on:
       - main
 
 env:
-  JAVA_VERSION: "17"
-  CONTAINER_REGISTRY_URL: "ghcr.io/infonl"
-  APPLICATION_NAME: "zaakafhandelcomponent"
+  JAVA_VERSION: '17'
```

Comment:
Sorry about this.. but I just noticed that in our other CodeQL workflow we use double quotes.. And that was more or less a default workflow generated by GitHub. So maybe double quotes is then more the standard then single quotes? i don't mind either way..